docs/getting-started.md

```diff
@@ -232,7 +264,706 @@ curl localhost:10101/index/repository/query \
+```
+
+Which repositories were starred by user 14 or 19:
+``` request
+response = client.query(repository.union(stargazer.row(14), stargazer.row(19)));
+System.out.println("User 14 or 19 starred: " + response.getResult().getRow().getColumns());
+```
+``` response
+User 14 or 19 starred: [1, 2, 3, 361, 362, 368, 376, 377, 378, 382, 386, 388, 391, 396, 398, 400, 409, 411, 412, 416, 426, 428, 430, 435, 436, 450, 452, 453, 454, 456, 460, 461, 464, 465, 466, 469, 470, 483, 484, 486, 487, 489, 490, 491, 500, 503, 504, 505, 512, 514]
+```
+
+Which repositories were starred by user 14 and 19 and also were written in language 1:
+``` request
+response = client.query(repository.intersect(stargazer.row(14), stargazer.row(19), language.row(1)));
+System.out.println("Both user 14 and 19 starred and were written in language 1: " + response.getResult().getRow().getColumns());
+```
+``` response
+Both user 14 and 19 starred and were written in language 1: [2, 362, 416, 461]
+```
+
+Set user 99999 as a stargazer for repository 77777:
+``` request
+client.query(stargazer.set(99999, 77777));
+System.out.println("Set user 99999 as a stargazer for repository 77777");
+```
+``` response
+Set user 99999 as a stargazer for repository 77777
+```
+
+Please note that while user ID 99999 may not be sequential with the other column IDs, it is still a relatively low number. 
+Don't try to use arbitrary 64-bit integers as column or row IDs in Pilosa - this will lead to problems such as poor performance and out of memory errors.
+
+For more information about java-pilosa, please see our Java client library at [java-pilosa](https://github.com/pilosa/java-pilosa) or checkout the java-pilosa [Data Model and Queries](https://github.com/pilosa/java-pilosa/blob/master/docs/data-model-queries.md) section for more query options.
+
+#### Python Users
+
+Pilosa requires Python 2.7 or higher or Python 3.4 or higher.
+
+##### Create the Environment
+
+Create a new project folder:
+```
+mkdir getting-started && cd getting-started
+```
+In this folder, we will download two CSV files to provide data to our fields later on. Download the `stargazer.csv` and `language.csv` files here:
+```
+curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/stargazer.csv
+curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/language.csv
+```
+We will also download two text files. One is the `requirements.txt` that will install python-pilosa later on and the other is `languages.txt` which will provide context to the `language` field.
+```
+curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/python/requirements.txt
+curl -O https://raw.githubusercontent.com/pilosa/getting-started/master/language.txt
+```
+We will now create the python environment:
+```
+python3 -m venv startrace
+```
+
+Next, we activate the python environment we created and install the requirements (and python-pilosa):
```

Comment:
no need for the "and" in "and python-pilosa". the requirements file contains only a single dependency, which is the python package called "pilosa", implemented in the repository named "python-pilosa".

Reply:
So should it be:
Next we activate the python environment we created and install the requirements for python-pilosa.
?

Reply:
"Next, we activate the python environment we created and install the single dependency, python-pilosa"

or

"Next, we activate the python environment we created and install python-pilosa"

The requirements are **for** the current project, getting-started. The requirements **are** a list of dependencies, in this case including only one item.